### DISCUSSION TITLE
Assignment 5

### DISCUSSION THREAD

**Discussion by hodgemi** [ebajcar/web10199_worksheets#28] [Q&A]
Assignment 5 is not loading for me, is anybody else having this issue?

**Comment by ebajcar** [comment]
[just posted info](https://teams.microsoft.com/l/message/19:skJhEOjB_Cp9kxPWsNekRSdVJ69RpqY7q5fu-d_jRiA1@thread.tacv2/1626123478294?tenantId=465ac757-2131-4711-b9a3-d8278b5c0b14&groupId=d908b0a0-4b11-46cd-8024-c8f772340868&parentMessageId=1626123478294&teamName=TU1-1215-42335-SYST10199-Web%20Programming&channelName=General&createdTime=1626123478294)

**Comment by Dev-999Sheridan** [comment]
I have having issue in assignment 5. When I clicked "Start Quiz" it showed me page with no questions and just "Submit Quiz" box . I can share Screenshot of the same if needed and I had also recorded my screen when i tried another time so i can share that too is needed. 
![2021-07-18](https://user-images.githubusercontent.com/84543113/126086781-1e5ffea7-efe7-4b96-85c8-1b5589c882a8.png)
